### ISSUE TITLE
Substance.summary returns empty string

### ISSUE THREAD

**Issue by dlbnco** [psychonautwiki/bifrost#13]
Most of not all substances are returning the `summary` field as an empty string.

# Example

## `Query.substances`
https://api.psychonautwiki.org/?query=query%7Bsubstances(query:%22MDMA%22)%7Bname%20summary%7D%7D
```graphql
{
  substances(query: "MDMA") {
    name
    summary
  }
}

```
```json
{
  "data": {
    "substances": [
      {
        "name": "MDMA",
        "summary": ""
      }
    ]
  }
}
```

**Comment by KenanSulayman** [comment]
Yes, the summaries are currently not implemented. No substance is returning a summary: https://api.psychonautwiki.org/?query=query%7Bsubstances(limit:5000)%7Bname%20summary%7D%7D.

I plan on eventually using Parsoid to provide programmatic text access to the articles. The summaries must either be provided by an elaborate NLP library (from the other text of the article) or by our users. The latter is a significant effort, so I'm planning this carefully.

**Comment by dlbnco** [comment]
>The summaries must either be provided by an elaborate NLP library (from the other text of the article)

Should be easier now with GPT-3 :)

**Comment by 19h** [comment]
At this point GPT-3 is nothing but a massive PR stunt for OpenAI being able to start billing enterprise customers for usage of their API. There's documentation on what GPT-3 is, but there's no place beyond OpenAI itself where this is usable.

**Comment by razorstorm** [comment]
Would it make sense for the summary field to simply return the intro text for the substance? (i.e. the first few paragraphs on the article before the table of contents)
